packages/react-dom/src/events/__tests__/SyntheticEvent-test.js

```diff
@@ -245,17 +243,15 @@ describe('SyntheticEvent', () => {
     expect(expectedCount).toBe(1);
   });
 
-  // TODO: reenable this test. We are currently silencing these warnings when
-  // using TestUtils.Simulate to avoid spurious warnings that result from the
-  // way we simulate events.
-  xit('should properly log warnings when events simulated with rendered components', () => {
+  it('should properly log warnings when events simulated with rendered components', () => {
     let event;
     const element = document.createElement('div');
+    document.body.appendChild(element);
     function assignEvent(e) {
       event = e;
     }
     const node = ReactDOM.render(<div onClick={assignEvent} />, element);
-    ReactTestUtils.Simulate.click(ReactDOM.findDOMNode(node));
+    ReactDOM.findDOMNode(node).click();
```

Comment:
Can remove findDOMNode here too. Actually we should remove it in a lot of places in the tests, it's often completely unnecessary and is confusing